src/ml/neural_net/mps_layer_instance_norm_data_loader.mm

```diff
@@ -0,0 +1,202 @@
+#include <ml/neural_net/mps_layer_instance_norm_data_loader.h>
+#include <ml/neural_net/mps_weight.h>
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
Where are these two C headers used?

I also don't see any C++ in this file. Could we leave a TODO here that this file can become a .m file once `mps_weight.h` stops using C++?

Reply:
removed the c++ header files.